crates/storage/db/src/implementation/mdbx/mod.rs

```diff
@@ -61,9 +63,11 @@ impl DatabaseEnvKind {
 
 /// Arguments for database initialization.
 #[derive(Clone, Debug, Default)]
-pub struct DatabaseArguments {
+pub struct DatabaseArguments<R: RangeBounds<usize> + Clone = Range<usize>> {
     /// Client version that accesses the database.
     client_version: ClientVersion,
+    /// Database geometry settings. If [None], default values will be used.
+    geometry: Option<Geometry<R>>,
```

Comment:
imo, we should just initialize this with the defaults and not make it optional

Reply:
fixed, thanks